app/helpers/search_helper.rb

```diff
@@ -85,6 +85,20 @@ def ordering
     end
   end
 
+  # Transforms controller params from get request into query args for a search
+  # @return [Hash]
+  def get_query_args
```

Comment:
How is this different from the query_args method? Is there something missing there?